README.md

```diff
@@ -40,12 +40,18 @@ jobs:
       - name: Comment Webpage Screenshot
         uses: opengisch/comment-pr-with-images@upload_only
         with:
-          # Optional, the action will create a new branch and
+          # required: Comma seperated list of files to upload.
+          images: "tests_artifacts/**/*.gif, tests_artifacts/**/*.png"
+          # optional: The action will create a new branch and
           # upload the screenshots to that branch.
           upload_to: github_branch  # Or, imgur
-          # Comma seperated list of files to upload.
-          images: "tests_artifacts/**/*.gif, tests_artifacts/**/*.png"
-          # Optional
+          # optional (defaults to the text used in this example): The message to use in the comment
+          custom_attachment_msg: "Screenshot for commit..."
+          # optional (defaults to `true`): Whether the previous comment should be edited with
+          # a deprecation warning along with a reference to the last comment
+          edit_previous_comment: true
```

Comment:
Can you add new inputs also in the table above (under Workflow inputs) ? Or if you don't like duplication, remove the table above altogether.